containers/layouts/StickyLabelsToggle.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Toggle, useToggle } from 'react-components';
+import { STICKY_LABELS } from 'proton-shared/lib/constants';
+
+const { ON, OFF } = STICKY_LABELS;
+
+const StickyLabelsToggle = ({ id, stickyLabels, onToggle, loading, ...rest }) => {
+    const { state, toggle } = useToggle(stickyLabels === ON);
+
+    const handleToggle = () => {
```

Comment:
Use `event` coming as parameter from `handleToggle` to get the value instead of using the `state` variable please.

Reply:
:+1: